docker-compose.yml

```diff
@@ -31,3 +31,14 @@ services:
     depends_on:
       - standalone-chrome
     command: 'npm run wdio-default'
+
+  test-ci-lowlight:
```

Comment:
Lets remove this custom service and update the travis commands to run:


```
script: travis_wait docker-compose run -e FORM_FACTOR=enormous test-ci -sh 'npm run wido-lowlight'

```

Reply:
Addressed it here : https://github.com/cerner/terra-framework/pull/978/commits/57fd4a3318250ef2f905377a4bcd0f77c32bbab3